internal/tools/tool/base.go

```diff
@@ -0,0 +1,115 @@
+package tool
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+type Base struct {
+	name              string
+	isInstalledCalled bool
```

Comment:
>isInstalledCalled bool [](http://example.com/codeflow?start=1&length=22)

you could change `installed` to a `*bool` to avoid a separate flag

Reply:
Good feedback, fixed